app/Traits/SortCategories.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Traits;
+
+use Illuminate\Support\Facades\Auth;
+use App\Models\Category;
+
+trait SortCategories
+{
+    protected function sort($query, $categories)
+    {
+        switch($query['sortBy']){
+            case 'Name': 
+                $sortBy = 'name';
+                break;
```

Comment:
is this still needed? we are already using strtolower which will make all the string lowercase. can you double check?